lms/djangoapps/commerce/api/v1/models.py

```diff
@@ -12,17 +12,21 @@
 
 log = logging.getLogger(__name__)
 
+UNDEFINED = object()
```

Comment:
There must be a reason you are using this instead. Can you explain again why `None` won't suffice?

Reply:
If a Course object's `verification_deadline` is set to `None`, the call to `VerificationDeadline.set_deadline()` inside `Course.save()` will delete the verification deadline for the course.  This is a situation where we need three general states:
1) The deadline is explicitly set to a date, so we should create or update the deadline for this course.
2) The deadline is explicitly set to None, so we should delete the deadline.
3) The deadline is undefined, so we should take no action on the deadline record.